Kentico.Kontent.Management/Models/Shared/ListingResponseModel.cs

```diff
@@ -32,19 +32,13 @@ public async Task<IListingResponseModel<T>> GetNextPage()
             return new ListingResponseModel<T>(_nextPageRetriever, nextPage.Pagination?.Token, _url, nextPage);
         }
 
-        public bool HasNextPage()
-        {
-            return _continuationToken != null;
-        }
+        public bool HasNextPage() => 
```

Comment:
Nitpick: this could be on one line

Reply:
I meant `public bool HasNextPage() => _continuationToken != null;` but it does not matter :).